### PR TITLE
re-review notebook 2

### PR DESCRIPTION
**Pull request by mco-gh**

### CODE REVIEW

File: notebooks/2_model_training_custom_xgboost-experiment.ipynb [hunk 1]
```diff
@@ -53,19 +53,21 @@
    "source": [
```
Comment (saeedaghabozorgi) [inline notebooks/2_model_training_custom_xgboost-experiment.ipynb:53]: <p>It sounds good 👍 </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mco-gh/fraudfinder/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (saeedaghabozorgi) [inline notebooks/2_model_training_custom_xgboost-experiment.ipynb:53]: <p>I agree that we should limit our support only for Colab and Workbench.</p><p>Having a setup-instruction in place would be resolved it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mco-gh/fraudfinder/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (saeedaghabozorgi) [inline notebooks/2_model_training_custom_xgboost-experiment.ipynb:53]: <p>Can be removed to be replaced by setup up notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mco-gh/fraudfinder/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (saeedaghabozorgi) [inline notebooks/2_model_training_custom_xgboost-experiment.ipynb:53]: <p>Agreed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mco-gh/fraudfinder/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (saeedaghabozorgi) [inline notebooks/2_model_training_custom_xgboost-experiment.ipynb:53]: <p>Agreed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mco-gh/fraudfinder/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (saeedaghabozorgi) [inline notebooks/2_model_training_custom_xgboost-experiment.ipynb:53]: <p>Makes sense. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mco-gh/fraudfinder/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (saeedaghabozorgi) [inline notebooks/2_model_training_custom_xgboost-experiment.ipynb:53]: <p>Yes, it should be interpolated directly. Makes it much easier to read the code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mco-gh/fraudfinder/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (saeedaghabozorgi) [inline notebooks/2_model_training_custom_xgboost-experiment.ipynb:53]: <p>Maybe we can add more nodes to the Feature Store for serving to make it faster. I will investigate it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mco-gh/fraudfinder/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Comment (saeedaghabozorgi) [inline notebooks/2_model_training_custom_xgboost-experiment.ipynb:53]: <p>It is failed or it is a warning ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mco-gh/fraudfinder/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Comment (saeedaghabozorgi) [inline notebooks/2_model_training_custom_xgboost-experiment.ipynb:53]: <pre class="ql-syntax" spellcheck="false">Line #26.    # /opt/conda/lib/python3.7/site-packages/sklearn/metrics/_classification.py:1308: UndefinedMetricWarning: Precision and F-score are ill-defined and being set to 0.0 in labels with no predicted samples. Use <code>zero_division</code> parameter to control this behavior.
</pre><p>I cannot reproduce it, are you running it in the Fraud Funder project?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mco-gh/fraudfinder/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>

Comment (mco-gh) [reply notebooks/2_model_training_custom_xgboost-experiment.ipynb]: <p>fixed in branch</p>

Comment (mco-gh) [reply notebooks/2_model_training_custom_xgboost-experiment.ipynb]: <p>removed, deferred to notebook 0</p>

Comment (mco-gh) [reply notebooks/2_model_training_custom_xgboost-experiment.ipynb]: <p>agree, removed</p>

Comment (mco-gh) [reply notebooks/2_model_training_custom_xgboost-experiment.ipynb]: <p>removed</p>

Comment (mco-gh) [reply notebooks/2_model_training_custom_xgboost-experiment.ipynb]: <p>fixed</p>

Comment (mco-gh) [reply notebooks/2_model_training_custom_xgboost-experiment.ipynb]: <p>cool, I'll remove this comment for now assuming you'll track this</p>

Comment (mco-gh) [reply notebooks/2_model_training_custom_xgboost-experiment.ipynb]: <p>Not sure. I'm due for a retest with the new setup strategy so I'm removing this todo and will let you know if I see this again and whether it seems like a warning or an error.</p>

Comment (mco-gh) [reply notebooks/2_model_training_custom_xgboost-experiment.ipynb]: <p>yes, on ff project but, per above, need to retest so I'll remove this and see if I can reproduce it later today.</p>

**Pull request merged**